docs/4.0-migration-guide/rng-removal.md

```diff
@@ -222,4 +240,5 @@ mbedtls_ssl_ticket_context;`
 
 ### Removal of `mbedtls_ssl_conf_rng`
 
-`mbedtls_ssl_conf_rng` has been removed from the library as it's sole purpose is to configure RNG for ssl and this is no longer required.
+`mbedtls_ssl_conf_rng` has been removed from the library as its sole purpose is to configure RNG for ssl and this is no longer required.
+```
```

Comment:
These closing backticks should be moved to above 'Removal of `mbedtls_ssl_conf_rng`' heading as they close the `mbedtls_ssl_ticket_context` block of code above.